libsawtooth/src/client/mod.rs

```diff
@@ -109,3 +117,9 @@ pub struct SingleState {
     pub data: Vec<u8>,
     pub head: String,
 }
+#[derive(Debug)]
+pub struct Status {
+    pub id: String,
+    pub invalid_transactions: Vec<Transaction>,
```

Comment:
The contents of this are not transactions but an `InvalidTransactionResult`  In the openapi spec, they look like this:
```
            properties:
              id:
                type: string
                example: 540a6803971d1880ec73a96cb97815a95d374cbad5d865925e5aa0432fcf1931539afe10310c122c5eaae15df61236079abbf4f258889359c4d175516934484a
              message:
                type: string
                example: Verb is \"inc\" but name \"foo\" not in state
              extended_data:
                type: string
                format: byte
                example: ZXJyb3IgZGF0YQ==
```

Reply:
Fixed